DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -433,6 +433,7 @@ public IActionResult SelfAssessmentOverview(int selfAssessmentId, string vocabul
 
             var optionalCompetencies = selfAssessmentService.GetCandidateAssessmentOptionalCompetencies(selfAssessmentId, delegateUserId);
             selfAssessmentService.UpdateLastAccessed(assessment.Id, delegateUserId);
+            selfAssessmentService.RemoveSignoffRequests(assessment.Id, delegateUserId);

```

Comment:
this is the landing page when you launch any self-assessment. The changes are to be done when you click send a request for confirming proficiencies (Request proficiency confirmation)

Reply:
Done